doc/api/http.md

```diff
@@ -729,8 +729,10 @@ This function is asynchronous. `callback` will be added as a listener for the
 
 Returns `server`.
 
-*Note*: The `server.listen()` method may be called multiple times. Each
-subsequent call will *re-open* the server using the provided options.
+*Note*:
+The `server.listen()` method may be called one more time iff there was an error
+during the first *listen* call or you explicitly called `server.close()`.
+Otherwise, `EALREADY` error would be emitted.
```

Comment:
"Otherwise, an `EALREADY` error is emitted." - likewise below.

Also, I'd probably replace "may be called one more time " with "can be called again."